cheatsheets/Docker_Security_Cheat_Sheet.md

```diff
@@ -30,7 +30,7 @@ volumes:
 
 ### RULE \#2 - Set a user
 
-Configuring container, to use unprivileged user, is the best way to prevent privilege escalation attacks. This can be accomplished in three different ways:
+Configuring container to use unprivileged user, is the best way to prevent privilege escalation attacks. This can be accomplished in three different ways as follows:
```

Comment:
```suggestion
Configuring the container to use an unprivileged user is the best way to prevent privilege escalation attacks. This can be accomplished in three different ways as follows:
```